app/Notification.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import './_ciq-notification.scss';
+
+class Notification extends React.Component {
+    constructor(props) {
```

Comment:
I think we can assert that `Notification.jsx` would not work without a `ChartNotifier`, so we can assign the event handlers in the constructor, instead of assigning it in the `render` method.